superset-frontend/src/dashboard/components/nativeFilters/FilterBar/index.tsx

```diff
@@ -147,6 +148,10 @@ const FilterBar: React.FC<FiltersBarProps> = ({
   const canEdit = useSelector<RootState, boolean>(
     ({ dashboardInfo }) => dashboardInfo.dash_edit_perm,
   );
+  const user: UserWithPermissionsAndRoles = useSelector<
+    any,
```

Comment:
```suggestion
    RootState,
```

Reply:
Thank you so much for catching this, @michael-s-molina!